build.gradle

```diff
@@ -127,3 +127,41 @@ configurations {
 }
 
 apply from: "$rootDir/gradle/gradle-mvn-mpp-push.gradle"
+
+allprojects {
+    tasks.register('publishMac') {
```

Comment:
Ported this from Stately. I like having these associated at the gradle level, instead of at the build script level the way I currently have it in MPS

Reply:
I like them as-is because I think it makes it clearer how to duplicate what the build script is doing locally, if we need to troubleshoot something.